pontoon/base/models.py

```diff
@@ -2911,7 +2911,13 @@ def for_project_locale(
 
     @classmethod
     def map_entities(
-        cls, locale, preferred_source_locale, entities, visible_entities=None
+        cls,
+        locale,
+        preferred_source_locale,
+        entities,
+        visible_entities=None,
+        sibling=False,
```

Comment:
Nit: We usually prefix boolean variables and function names with "is" or "has", so `is_sibling` would be better: https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig